docs/documentation/cli.md

```diff
@@ -1,67 +1,54 @@
 ---
 layout: default
-title: CLI
+title: Soda SQL CLI commands
 parent: Documentation
 nav_order: 2
 ---
 
-# Soda's Command Line Interface (CLI)
+# Soda SQL CLI commands
 
-> See our [Installation guide]({% link getting-started/installation.md %}) on how to install the `soda` command.
+| Command | Description |
+| ------- | ----------- |
+| `soda analyze` | Analyzes the contents of your warehouse and automatically prepares a scan YAML file for each table. Soda SQL puts the YAML files in the `/tables` directory inside the warehouse directory. See [Create a scan YAML file]({% link documentation/scan.md %}#create-a-scan-yaml-file) for details.|
+| `soda create` | Creates a new `warehouse.yml` file and prepares credentials in your `~/.soda/env_vars.yml`. Soda SQL does not overwrite or remove and existing environment variables, it only adds new. See [Create a warehouse YAML file]({% link documentation/warehouse.md %}#create-a-warehouse-yaml-file) for details. |
+| `soda scan` | Uses the configurations in your scan YAML file to prepare, then run SQL queries against the data in your warehouse. See [Run a scan]({% link documentation/scan.md %}#run-a-scan) for details. |
 
-The soda command line tool helps you to get started with
-your Soda SQL configuration files and run Soda Scans.
+## List of commands
 
-To see the list of available commands, enter `soda` in your terminal:
+To see a list of Soda SQL command-line interface (CLI) commands, use the `soda` command. 
 
+Command:
+```shell
+soda
 ```
-$ soda
+
+Output:
+```shell
 Usage: soda [OPTIONS] COMMAND [ARGS]...
 
-  Soda CLI version {version}
+  Soda CLI version 2.x.xxx
 
 Options:
   --help  Show this message and exit.
 
 Commands:
-  create  Creates a new warehouse.yml file and prepares credentials in your...
-  init    Finds tables in the warehouse and creates scan YAML files based
-          on...
+  analyze  Analyzes tables in the warehouse and creates scan YAML files...
+  create   Creates a new warehouse.yml file and prepares credentials in
+           your...
 
-  scan    Computes all measurements and runs all tests on one table.
+  init     Renamed to `soda analyze`
```

Comment:
We probably will remove this `init` soon as the name change is already there for a while.  So it might be good to remove it from the docs already.